src/Zikula/CoreBundle/Helper/PersistedBundleHelper.php

```diff
@@ -120,14 +120,15 @@ private function extensionIsActive(Connection $conn, string $class, string $type
      */
     public function addAutoloaders(ZikulaHttpKernelInterface $kernel, array $autoload = []): void
     {
+        $srcDir = dirname(dirname(dirname(__DIR__))) . '/';
```

Comment:
this feel odd to me. what about using `$kernel->get ProjectDir()` again? Does this produce a full or relative path?

Reply:
you are right; added usage of `getProjectDir()` in df8ab0cfe0f31a446451f3f23e29f317155bff24